app/src/App.vue

```diff
@@ -7,17 +7,19 @@ import { useUserDataStore } from '@/stores/user'
 
 import Drawer from '@/components/TheDrawer.vue'
 import NavBar from '@/components/NavBar.vue'
-import EditUserModal from '@/components/modals/EditUserModal.vue'
 import ToastContainer from '@/components/ToastContainer.vue'
+import ModalComponent from '@/components/ModalComponent.vue'
+import EditUserForm from '@/components/modals/EditUserForm.vue'
 
-import { isAddress } from 'ethers'
 // import { useDark, useToggle } from '@vueuse/core'
 import { useTipsBalance, useWithdrawTips } from './composables/tips'
 import { useErrorHandler } from './composables/errorHandler'
 import { useCustomFetch } from './composables/useCustomFetch'
 const { addErrorToast, addSuccessToast } = useToastStore()
 
 const toggleSide = ref(true)
+const toggleModal = ref(false)
```

Comment:
`toggleModal` sounds like an action or rather a function. I feel that showModal would be more clear.